mongodbatlas/data_source_mongodbatlas_projects.go

```diff
@@ -116,13 +135,18 @@ func flattenProjects(ctx context.Context, conn *matlas.Client, projects []*matla
 				fmt.Printf("[WARN] error getting project's teams assigned (%s): %s", project.ID, err)
 			}
 
+			apiKeys, err := getProjectApiKeys(conn, ctx, project.OrgID, project.ID)
+			if err != nil {
+				fmt.Printf("[WARN] error getting project's api keys (%s): %s", project.ID, err)
+			}
```

Comment:
is it okey to continue if you cannot find the project api keys?

Reply:
in some cases there might be no api key in the project, example: API key defined in the terraform provider has `ORG_OWNER` permission, after creating a project with `mongodbatlas-project `, this key will be assigned to the project with `GROUP_OWNER`. If I manually unassign this key from the project I still will be able to modify `mongodbatlas-project ` attributes like `teams`, as the key has `ORG_OWNER`, but no project level api keys
`getProjectApiKeys` uses this API https://docs.atlas.mongodb.com/reference/api/projectApiKeys/get-all-apiKeys-in-one-project/ and list project level api keys only